pkg/workceptor/kubernetes.go

```diff
@@ -683,6 +701,7 @@ func (kw *kubeUnit) SetFromParams(params map[string]string) error {
 		{name: "secret_kube_pod", permission: kw.allowRuntimePod, setter: setString(&userPod)},
 		{name: "kube_verify_tls", permission: kw.allowRuntimeTLS, setter: setBool(&ked.KubeVerifyTLS)},
 		{name: "kube_tls_ca", permission: kw.allowRuntimeTLS, setter: setString(&ked.KubeTLSCAData)},
+		{name: "pod_pending_timeout", permission: kw.allowRuntimeAuth, setter: setString(&ked.PodPendingTimeout)},
```

Comment:
should be permission allowruntimeparams

Reply:
I have made the changes. I dont think that this line is required anymore.